### DISCUSSION TITLE
QPageScroller overlap footer

### DISCUSSION THREAD

**Discussion by DaleMckeown** [quasarframework/quasar#16475] [Ideas / Proposals]
The documentation for `QPageScroller` states:

> The QPageScroller component helps in placing DOM elements / components wrapped by it into a static position within the content area of your QPage, no matter where the user scrolls.

> The great advantage of this is that the elements wrapped by this component will never overlap the layout header, footer or drawer(s), even if those are not configured to be fixed. In the latter case, the position will be offset so that the overlap won’t occur. Try it out with a non-fixed footer for example. When user reaches bottom of screen and footer comes into view, the component will shift up so it won’t overlap with the footer.

Sometimes it is favourable to have the scroller display over the footer.

Even the Quasar docs implement a QPageScroller that overlaps the footer. This is achieved by not using a `QFooter`, element, but instead using a plain div and custom CSS.

![image](https://github.com/quasarframework/quasar/assets/1249673/92e83be2-3239-4a57-b4f6-494e2d2ecc71)

This isn't an ideal solution as it does not use the HTML semantic `footer` element.

Maybe we could have options to allow us to override the `QFooter` checks?

**Comment by darkopetreski** [comment]
I also need this option - to overlap it with the footer. There are a lot of cases where this is needed.
